src/Xamarin.Android.Build.Tasks/Tasks/ResolveAndroidTooling.cs

```diff
@@ -131,7 +134,18 @@ public override bool Execute ()
 			ApkSignerJar = Path.Combine (AndroidSdkBuildToolsBinPath, "lib", ApkSigner);
 			AndroidUseApkSigner = File.Exists (ApkSignerJar);
 
-			bool aapt2Installed = File.Exists (Path.Combine (AndroidSdkBuildToolsBinPath, Aapt2));
+			if (string.IsNullOrEmpty (Aapt2ToolPath)) {
+				var aapt2 = Path.Combine (MonoAndroidHelper.GetOSBinPath(), Aapt2);
+				if (File.Exists (aapt2))
+					Aapt2ToolPath = MonoAndroidHelper.GetOSBinPath();
+				else {
+					aapt2 = Path.Combine (AndroidSdkBuildToolsBinPath, Aapt2);
+					if (File.Exists (aapt2))
+						Aapt2ToolPath = AndroidSdkBuildToolsBinPath;
+				}
+			}
+
+			bool aapt2Installed = File.Exists (Path.Combine (Aapt2ToolPath, Aapt2));
```

Comment:
Just a question, can we just get rid of all the `aapt2` fallback logic? Should we even use `aapt2` from the Android SDK if we ship it?

If a developer _really_ wants to use a different `aapt2`, they can change `$(Aapt2ToolPath)`, right?